planning/behavior_path_planner/src/scene_module/avoidance/avoidance_module.cpp

```diff
@@ -424,10 +424,23 @@ bool AvoidanceModule::canYieldManeuver(const AvoidancePlanningData & data) const
 {
   // transit yield maneuver only when the avoidance maneuver is not initiated.
   if (helper_.isShifted()) {
+    RCLCPP_DEBUG(getLogger(), "avoidance maneuver already initiated.");
     return false;
   }
 
+  // prevent sudden steering.
+  const auto registered_lines = path_shifter_.getShiftLines();
+  if (!registered_lines.empty()) {
+    const size_t idx = planner_data_->findEgoIndex(path_shifter_.getReferencePath().points);
+    const auto to_shift_start_point = calcSignedArcLength(
+      path_shifter_.getReferencePath().points, idx, registered_lines.front().start_idx);
+    if (to_shift_start_point < helper_.getMinimumPrepareDistance()) {
```

Comment:
TODO: output debug info

Reply:
@rej55 I added debug info in https://github.com/autowarefoundation/autoware.universe/pull/5572/commits/0d8c9cdae82ed07891daaf4affa775129e92d542.